tutorials/3d/fps_tutorial/part_two.rst

```diff
@@ -53,7 +53,7 @@ Add the following code to ``AnimationPlayer_Manager.gd``:
         "Knife_equip":["Knife_idle"],
         "Knife_fire":["Knife_idle"],
         "Knife_idle":["Knife_fire", "Knife_unequip", "Knife_idle"],
-        "Knife_unequip":["Idle_unarmed"],
+        "Knife_unequip":["Idle_unarmed"]
```

Comment:
Kind of a moot point, since the extra comma does not do anything. That said, I have no problem removing it (and I do not think the change is worth updating the ZIP files)